### ISSUE TITLE
Apollo 3 Type Error with SubscriptionServer

### ISSUE THREAD

**Issue by berkaey** [apollographql/apollo-server#5851]
I'm trying to setup an Apollo 3 + Typescript project, when I follow the subscription example on docs, I'm getting these errors.

<img width="875" alt="Screen Shot 2021-10-31 at 13 38 06" src="https://user-images.githubusercontent.com/4539951/139578639-4a3a0c8d-d1da-4c1a-80d9-c2e389c07f54.png">

both execute and subscribe has different type on GraphQL package.

Btw, I'm importing those from `graphql`.

Any suggestions?

**Comment by glasser** [comment]
Hi @berkaey. It's definitely possible there's an error in our doc example that explains how to continue to use the unmaintained `subscriptions-transport-ws` package with Apollo Server 3; we don't automatically type-check the doc examples. However, it's hard to debug this based on a screen shot, especially one where the error obscures the code. Do you think you could demonstrate the issue via a full repository that fails to build (eg a git repo or codesandbox.io sandbox)?

**Comment by riginoommen** [comment]
I also face same issue when the graphql is upgrades to v16.x

https://github.com/graphql/graphql-js/pull/2301

with the latest graphq the support of execute and subscribe is depricated

**Comment by berkaey** [comment]
@glasser something like this helps?

```js
import { execute, GraphQLError, subscribe, TypeDefinitionNode } from "graphql";
import { SubscriptionServer } from "subscriptions-transport-ws";

// setup subscription server
    this.apolloSubscrtipions = SubscriptionServer.create(
      {
        schema,
        // @ts-ignore
        execute,
        // @ts-ignore
        subscribe,

        async onConnect(
          connectionParams: Object,
          webSocket: WebSocket,
          context: any,
        ) {
          if (this.onConnect) {
            await this.clientDidConnect(connectionParams, webSocket, context);
          }

          this.log.info("CONNECTED");

          return context;
        },

        async onDisconnect(webSocket: any, connectionContext: any) {
          if (connectionContext) {
            const context = await connectionContext.initPromise;
            await this.clientDidDisconnect(
              webSocket,
              connectionContext,
              context,
            );
            this.log.info("DISCONNECTED");
          }
        },
      },
      {
        server: this.httpServer,
        path: this.config.paths.graphqlSubscription,
      },
    );

```

I had to use // @ts-ignore.

**Comment by glasser** [comment]
My belief is that if you upgrade to the newly-released `subscriptions-transport-ws@0.11.0` it should resolve the issue with no changes needed to the example code or Apollo Server. Let me know if that turns out not to be true!
